bpo-43795: Note Stable ABI PEP in What's New (#26479)
python/cpython: Doc/whatsnew/3.10.rst

C API Changes
=============

PEP 652: Maintaining the Stable ABI
-----------------------------------

> The Stable ABI (Application Binary Interface) for extension modules or

Reviewer: Hm. The Stable ABI and C API in here?

Author: Well, the limited C API is also defined. I went with the PEP name to make things shorter.

embedding Python is now explicitly defined.

> C API and ABI stability guarantees and best practices for using the Stable ABI
> are described in :ref:`stable`.

Reviewer 2: Minor nit: when I first read this I was slightly confused and re-read a few times, maybe because of the many "and" s, I suggest wording it like this:
```suggestion
:ref:`stable` describes C API and ABI stability guarantees along with best
practices for using the Stable ABI.
```

Author: Thanks, that does read better!
